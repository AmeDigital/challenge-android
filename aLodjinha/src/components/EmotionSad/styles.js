import { StyleSheet } from 'react-native';

import material from '../../utils/MaterialDesign';

export default StyleSheet.create({
	container: {
		flex: 1,
		justifyContent: 'center',
		alignItems: 'center'
	},
	image: {
		resizeMode: 'contain',
		width: 150
	},
	description: {
		fontSize: 18,
		fontWeight: 'bold',
		color: material.colorDark
	}
});
