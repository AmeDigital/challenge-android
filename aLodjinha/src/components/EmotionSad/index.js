import React from 'react';
import { View, Image, Text } from 'react-native';

import styles from './styles';

export default function EmotionSad() {
	return (
		<View style={styles.container}>
			<Image source={require('../../images/triste.png')} style={styles.image} />
			<Text style={styles.description}>Por enquanto categoria disponivel</Text>
			<Text style={styles.title}>"Games"</Text>
		</View>
	);
}
