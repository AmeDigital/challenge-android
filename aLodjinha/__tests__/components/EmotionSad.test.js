import React from 'react';

import renderer from 'react-test-renderer';

import EmotionSad from '../../src/components/EmotionSad';

describe('Component EmotionSad', () => {
	test('should render without erros', () => {
		const tree = renderer.create(<EmotionSad />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
