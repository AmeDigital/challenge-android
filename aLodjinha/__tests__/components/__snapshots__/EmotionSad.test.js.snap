// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component EmotionSad should render without erros 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/images/triste.png",
      }
    }
    style={
      Object {
        "resizeMode": "contain",
        "width": 150,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#2d3142",
        "fontSize": 35,
        "fontWeight": "bold",
      }
    }
  >
    Produtos indispon√≠veis no momento
  </Text>
</View>
`;
